lib/src/ktfmt/java/com/diffplug/spotless/glue/ktfmt/KtfmtFormatterFunc.java

```diff
@@ -59,11 +57,8 @@ public String apply(@Nonnull String input) throws Exception {
 	private FormattingOptions createFormattingOptions() throws Exception {
 		FormattingOptions formattingOptions;
 		switch (style) {
-		case DEFAULT:
-			formattingOptions = new FormattingOptions();
-			break;
-		case DROPBOX:
-			formattingOptions = Formatter.DROPBOX_FORMAT;
+		case META:
+			formattingOptions = Formatter.META_FORMAT;
```

Comment:
`0.51.0` or newer can access this, old versions will throw `IllegalStateException`. If we have to compact previous versions, have to do something like 

https://github.com/diffplug/spotless/blob/3a0e5bc0013c23c6d8e0562fd62b5087dcdb8cc6/lib/build.gradle#L109-L125

Reply:
Older versions will not get here as it is forked into using `KtfmtStep.KtfmtFormatterFuncCompat` as you can see on line 213 from the `KtfmtStep.java` files

Reply:
Sorry, my bad. :(